packages/espree/lib/options.js

```diff
@@ -3,11 +3,15 @@
  * @author Kai Cataldo
  */
 
+/**
+ * @import { ecmaVersion, ParserOptions } from "../espree.js";
```

Comment:
`ecmaVersion` looks unused.

```suggestion
 * @import { ParserOptions } from "../espree.js";
```